assignment02/js/script.js

```diff
@@ -1,40 +0,0 @@
-$(document).ready();

```

Comment:
You are checking here for the document to be ready - but you haven't defined a function to occur when that happens. All the rest of your code should be in that function.